### ISSUE TITLE
Add support for <amp-accordion>

### ISSUE THREAD

**Issue by amedina** [Automattic/amp-wp#730]
Check: [<amp-accordion>](https://www.ampproject.org/docs/reference/components/amp-accordion)
Used often in conjunction with [<amp-sidebar](https://www.ampproject.org/docs/reference/components/amp-sidebar) for implementing menus and navigation

**Comment by westonruter** [comment]
By adding support, do you mean to automatically enqueue the script in the head when it is used in the page?

**Comment by westonruter** [comment]
Or rather, support in that the bundled theme template should go ahead and display widgets inside of an `amp-accordion`?

**Comment by shangdev** [comment]
@westonruter 
Does that mean you need to manually import amp-sidebar scripts in the theme?

**Comment by westonruter** [comment]
@shangdev No. If you're using AMP theme support, then you don't need to import any scripts. If you're using legacy AMP mode, then you'll only need to import the `amp-sidebar` script if the element appears outside the content.
